### DISCUSSION TITLE
Abuout  "Click to Resume..."

### DISCUSSION THREAD

**Discussion by liqijian101** [luc-github/ESP3D#844] [F.A.Q] [answered]
When I use the serial port (Extra SD) to print, the LCD screen displays "Click to Resume...". How can I turn it off and print directly.

**Comment by luc-github** [comment]
this come from your printer FW: Marlin ?
You may have look here : https://github.com/luc-github/ESP3D/issues/373#issuecomment-696079532

**Reply by luc-github** [reply]
@liqijian101 did it solved your issue ?

**Reply by liqijian101** [reply, marked as answer]
new testing.
If I turn off the polling function, I can avoid this phenomenon

**Reply by luc-github** [reply]
looks like Marlin configuration issue - as ESP3D only send GCODE commands to print, then commands you defined in polling commands
What is the polling delay ?

**Reply by liqijian101** [reply]
3000ms

**Reply by luc-github** [reply]
and what are the polling commands ?

**Reply by liqijian101** [reply]
M105

**Reply by luc-github** [reply]
does same issue happen when you send manually M105 when print and polling is disabled ?

**Reply by liqijian101** [reply]
I will test it
